2-mandatory/1-writers.js

```diff
@@ -71,6 +72,11 @@ Exercise 2:
 */
 
 function logDeadWritersInTheirForties() {
+  writers.forEach(o => {
+    if (o.age > 40 && o.age < 50 && o.alive === false){
```

Comment:
Hello Deago
Your If statement needs to be modified because it does not include someone who is age 40
I suggest
(writer.age >= 40 && writer.age <= 49 && writer.alive === false)